lib/aruba/environment/windows_environment.rb

```diff
@@ -0,0 +1,11 @@
+require 'aruba/environment/basic_environment'
+
+module Aruba
+  class WindowsEnvironment < BasicEnvironment
+    def initialize
```

Comment:
Sounds good.  (And your English really, really good.  I think most of my questions are because I just don't know the way you think -- no matter which language. :-) )